salt/modules/timezone.py

```diff
@@ -350,7 +350,7 @@ def zone_compare(timezone):
     if "Solaris" in __grains__["os_family"] or "AIX" in __grains__["os_family"]:
         return timezone == get_zone()
 
-    if "FreeBSD" in __grains__["os_family"]:
+    if __grains__["os_family"] in {"Arch", "FreeBSD"}:
```

Comment:
**suggestion**: FWIW a tuple `(...)` is typically preferred over a set `{...}` for this kind of comparison:

```suggestion
    if __grains__["os_family"] in ("Arch", "FreeBSD"):
```

though looking at the code around it, would it be more appropriate to do

```
if "Arch" in __grains__["os_family"] or "FreeBSD" in __grains__["os_family"]:
```
? It seems like at least the other checks are thinking that `os_family` could be a list-type, or that they might contain more than just say "FreeBSD" but might have something like "Cool FreeBSD" or something to that effect.